Makefile

```diff
@@ -598,8 +598,9 @@ YACC		:= bison
 LEX     	:= flex
 PATCH		:= patch
 GZIP		:= gzip
-TAR		:= tar
+TAR			:= tar
 UNZIP		:= unzip -qq -u
+GIT			:= git
```

Comment:
Could you keep the same indentation level?

Reply:
Yes.